### ISSUE TITLE
[2.4] Monitoring does not work for Windows Server Core worker nodes

### ISSUE THREAD

**Issue by aiyengar2** [rancher/rancher#31141]
Backport new Monitoring charts (v0.1.5, v0.2.1) introduced in https://github.com/rancher/rancher/issues/27911 that have new wmi_exporter-package versions to support Windows Server Core nodes

**Comment by thehejik** [comment]
Rancher v2.4 doesn't support Windows Server 2004. 1909 should be supported - trying Windows_Server-1909-English-Core-ContainersLatest-2021.01.13 `ami-008ec03d9035c8ee7`

**Comment by aiyengar2** [comment]
@thehejik Monitoring 20H2 and 2004 should be supported since we are backporting the changes in https://github.com/rancher/rancher/issues/31337 as well.

It requires the version of wmi_exporter-package to be v0.0.7.

**Comment by thehejik** [comment]
@aiyengar2 I see, but `rancher/rancher-agent:2.4.x` itself doesn't support them (according to Sowmya only 1809,1903 and 1909 are supported in rancher v2.4) so I'm unable to deploy windows 2004 nor 20H2 workers with rancher-v2.4.x. Some info https://github.com/rancher/rancher/issues/25706

I was trying deploy a win 2004 worker node by `rancher:v2.4.14-rc1` and docker failed on:
```
docker: Error response from daemon: hcsshim::CreateComputeSystem f4b0a52dd914f00d3005fc127b8982c2d1feae016532d6160b55923b963ecd4d: The container operating system does not match the host operating system.
```
Windows 1909 works normally...

**Comment by thehejik** [comment]
Rancher `v2.4.14-rc1` / UI `v2.4.46`

The `exporter-node-windows-cluster-monitoring` DaemonSet from v1 monitoring chart 0.2.1 as well as 0.1.5 is using new released image `rancher/wmi_exporter-package:v0.0.7`

Deployed windows cluster with kubernetes `v1.18.16-rancher1-1`, flannel with VXLAN - 1 etcd, 1 control plane and 1 linux worker nodes, 3 worker nodes Windows_Server-1909-English-Core-ContainersLatest-2021.01.13 `ami-008ec03d9035c8ee7` (us-east-2)

Following was successfully tested on both chart versions:
* prometheus targets UP
* Graphs in Amber UI (cluster wide, nodes)
* all resources Ready
